js/data.js

```diff
@@ -6,3 +6,8 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+var previousData = localStorage.getItem('data model');
```

Comment:
Don't have spaces in your localStorage property name. Use a `-` to delimit your words